packages/manager/apps/dedicated/client/app/ip/ip/block/move/ip-ip-block-move.controller.js

```diff
@@ -142,6 +156,33 @@ export default /* @ngInject */ ($scope, $q, $translate, Ip, Alerter) => {
       serviceNameChoosed && (nextHopSelectedPCC() || nextHopSelectedOther())
     );
   };
+  $scope.cancelAction = function cancelAction() {
+    atInternet.trackClick({
+      name: `${$scope.data?.tracking}::cancel`,
+      type: 'action',
+    });
+    $scope.resetAction();
+  };
+  $scope.nextAction = function() {
+    atInternet.trackClick({
+      name: `${$scope.data?.tracking}::next`,
+      type: 'action',
+    });
+  };
+
+  $scope.previousAction = function() {
+    atInternet.trackClick({
+      name: `${$scope.data?.tracking}::back`,
+      type: 'action',
+    });
+  };
+  $scope.backAction = function backAction() {
+    atInternet.trackClick({
+      name: `${$scope.data?.tracking}::back`,
+      type: 'action',
+    });
+    return $scope.loading.save;
+  };
```

Comment:
i think we can prefix function with `on` to indicate that it's triggered by event.
`onCancelAction, onNextAction, onBackAction`.

Reply:
Taken care suggestions